keyboards/ungodly/priya50/info.json

```diff
@@ -0,0 +1,108 @@
+{
+    "manufacturer": "Ungodly Design",
+    "keyboard_name": "Priya50",
+    "maintainer": "Luis Godinez",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP24", "pin_b": "GP23"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP4", "GP6", "GP7", "GP8", "GP9", "GP13", "GP5", "GP3", "GP2", "GP11", "GP14", "GP15", "GP22"],
+        "rows": ["GP0", "GP1", "GP10", "GP17"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "brightness_steps": 8,
+        "led_count": 8,
+        "saturation_steps": 8,
+        "sleep": true
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x5544",
+        "vid": "0x3530"
```

Comment:
This Vendor ID is in use, already.  In fact, the 0x3000 block is being actively assigned, and should be avoided, unless you have explicit permission. 

https://www.the-sz.com/products/usbid/index.php?v=0x3530&p=&n=

Otherwise, please change to something that isn't in use. (generally, the further past 0x4000, the better).